modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+The *cluster-creating AWS account* user runs the following command in their terminal to create the cluster in the shared VPC.
+
+[NOTE]
```

Comment:
If you decide to adopt the revision that I suggested, you can remove this note, since it's been moved further up in the topic.